src/App.js

```diff
@@ -1,24 +1,27 @@
-import logo from './logo.svg';
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './Redux/store';
 import './App.css';
+import Navigation from './components/Navigation';
+import Rockets from './pages/Rockets';
+import Missions from './pages/Missions';
+import MyProfile from './pages/MyProfile';
+import Dragons from './pages/Dragons';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Provider store={store}>
```

Comment:
- [ ] You already have the `Provider` inside `index.js`. I think it is redundant here, kindly remove one of them. 🧐️

Reply:
@amejid Thank you for the suggestions